### ISSUE TITLE
comment support

### ISSUE THREAD

**Issue by geddski** [styled-components/vscode-styled-components#13]
oh man if this extension made it so `cmd+/` toggled CSS comments that would be so amazing:

```js
const HoodieStyle = styled.div`
  /*  width: 200px; */
  /*  height: 300px; */
  border: 5px solid black;
`;
```

**Comment by retrospct** [comment]
Second this enhancement. It's a decent quality of life improvement :D

**Comment by mjbvz** [comment]
I believe this was fixed by #33

**Comment by dan-dr** [comment]
This is not working for me... `Ctrl + /` yields:
```css
.ant-table-thead > tr > th.ant-table-column-sort {
    ${'' /* background: ${Colors.offBlack}; */}
  }
```
instead of 
```css
.ant-table-thead > tr > th.ant-table-column-sort {
    // background: ${Colors.offBlack};
  }
```

same for `Alt + Shift + A` for multiline comments
